Formula/macchina.rb

```diff
@@ -0,0 +1,20 @@
+class Macchina < Formula
+  desc "Basic system information fetcher, with a focus on performance and minimalism"
+  homepage "https://github.com/grtcdr/macchina"
+  url "https://github.com/grtcdr/macchina/archive/v0.6.0.tar.gz"
+  sha256 "bf6eaaca560fd2e7f04594abc6fc4615e6d3867a74816a26146f0244266cdffe"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    Dir.chdir "macchina" do
+      system "cargo", "install", *std_cargo_args
+    end
+    bin.install "target/release/macchina"
+  end
+
+  test do
+    system "macchina", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
> This looks like a nice piece of software, but it currently doesn't meet the requirements for inclusion in homebrew/core. The CI log reads
> 
> ```
> macchina:
>   * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
> Error: 1 problem in 1 formula detected
> ```
> 
> If you wish, you can host this formula in your own tap. Here are some docs to help you get started:
> 
> https://docs.brew.sh/Taps
> https://docs.brew.sh/Interesting-Taps-and-Forks
> https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
> https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

Oh ok, sorry I didn't know that. I'll see that I can do about hosting this in my own tap. Thank you, guys :)